visidata/features/dedupe.py

```diff
@@ -102,9 +102,21 @@ def _reload(self=vs):
     return vs
 
 
+@Sheet.api
+@asyncthread
+def hide_unvarying_cols(sheet):
+    if len(sheet.rows) < 2:
+        return
+    for col in sheet.visibleCols:
+        vals = (col.getTypedValue(r) for r in sheet.rows)
+        first = next(vals)
+        if all(v == first for v in vals):
+            col.hide()
+
 # Add longname-commands to VisiData to execute these methods
 BaseSheet.addCommand(None, "select-duplicate-rows", "sheet.select_duplicate_rows()", "select each row that is a duplicate of a prior row")
 BaseSheet.addCommand(None, "dedupe-rows", "vd.push(sheet.dedupe_rows())", "open new sheet in which only non-duplicate rows in the active sheet are included")
+BaseSheet.addCommand(None, 'hide-unvarying-cols', 'sheet.hide_unvarying_cols()', 'hide any column that has multiple rows but only one distinct value')
 
 vd.addMenuItems('''
```

Comment:
Let's add this command to an appropriate place in the menu.

Reply:
I think it'd be good to put it under `Column`, with `hide` and `unhide-cols`. I think the only way to do that is to do `addMenuItems` with the other hide commands in `features/layout.py`. Is that right? So then the function definition and `addCommand()` should move there.

Reply:
Menu items can be added anywhere, so if this is the best place for the code, then the addCommand and its associated Menu item should be here.